plugins/fabrik_element/fileupload/fileupload.php

```diff
@@ -457,11 +458,6 @@ protected function ajaxFileFilters()
 		$return = new stdClass;
 		$exts = $this->_getAllowedExtension();
 
-		foreach ($exts as &$ext)
```

Comment:
Except that the regex in that change wouldn't have picked up all uses of the dot, see comment on that line.  So for example ".foo, .bar,.baz, txt, .doc" would have left .baz as dotted, as well as .foo.  Which was why I did it here rather than in the regex in the first place, as RE's are just tricky, even simple ones.  So I deliberately did it in two steps, removing the spaces in the re, and the dots here.

This is why we tend to go on about "if it ain't broke, don't fix it" in PR's, and not including unrelated fixes in new feature PR's.  I'm not trying to be an ass, just pointing this out as a classic example.  The code as-is worked.  This fix for non-broken code broke it.  If you want to fix things unrelated to a new feature, submit those as separate little PR's, where they are easier to review and spot problems with.

This is the main reason some of your new feature PR's go un-merged, as they tend to have unrelated code changes, which we just don't have the time to comb through in addition to the meat of the change, so baby gets thrown out with the bathwater, as it were.